### DISCUSSION TITLE
add symbol on chart on specific pivot date

### DISCUSSION THREAD

**Discussion by qn222ad** [highfestiva/finplot#189] [Q&A] [answered]
Can I ask how can I plot date and at the same time with symbols like in picture?
Thanks
![2021-05-15 18_12_41-Window](https://user-images.githubusercontent.com/77920707/118387182-2bcb9080-b61d-11eb-9376-caa74ee29b1c.png)

**Comment by highfestiva** [comment, marked as answer]
See the example in the [README](https://github.com/highfestiva/finplot#example-2).
